internal/ts_repositories.bzl

```diff
@@ -34,6 +34,12 @@ def ts_setup_workspace():
         importpath = "github.com/kylelemons/godebug",
     )
 
+    go_repository(
```

Comment:
I wonder how this will work in google3... I'll probably have to do some stitching...